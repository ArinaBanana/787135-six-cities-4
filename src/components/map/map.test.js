import React from "react";
import renderer from "react-test-renderer";
import Map from "./map.jsx";

const markers = [
  {
    coordinates: [52.3909553943508, 4.85309666406198],
    color: `path`
  },
  {
    coordinates: [52.3809553943508, 4.939309666406198],
    color: `path`
  }
];

it(`Should render Map`, () => {
  const tree = renderer.create(<Map markers={markers} height={`600px`} />, {createNodeMock: () => {
    return document.createElement(`div`);
  }}).toJSON();

  expect(tree).toMatchSnapshot();
});
