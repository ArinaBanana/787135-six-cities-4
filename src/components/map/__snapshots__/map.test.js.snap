// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Map 1`] = `
<div
  style={
    Object {
      "height": "600px",
    }
  }
/>
`;
